experimental/http_logger/http_logger_test.go

```diff
@@ -48,6 +48,39 @@ func TestHTTPLogger(t *testing.T) {
 		_, err = os.Stat(f.Name())
 		require.Equal(t, true, errors.Is(err, os.ErrNotExist))
 	})
+
+	t.Run("should set path and enabled overrides", func(t *testing.T) {
```

Comment:
Maybe a small test for when `NewHTTPLogger` receives no `opts`?

Reply:
@wbrowne that should be covered by these two: https://github.com/grafana/grafana-plugin-sdk-go/blob/9429436d499a1b331af6d26ae59222fb1d320dfb/experimental/http_logger/http_logger_test.go#L18-L50

Reply:
Ah my bad! Thanks for clarifying 😄

Reply:
no prob at all. thanks for the review!